site/docs/v1/tech/apis/identity-providers/_saml-complete-user-registration.adoc

```diff
@@ -0,0 +1,4 @@
+The user does not need to exist yet in FusionAuth to utilize this API. The `SAMLResponse` returned from the SAML v2 provider will be used to retrieve the user's email address, if that user does not yet exist in FusionAuth the user will be created. If [field]#Create registration# has been enabled for this identity provider and the user does not yet have a registration for this application, a registration will be automatically created for the user. 
```

Comment:
consider 
email address, if => email address **_and_** if

Reply:
or consider
The user does not need to exist yet in FusionAuth to utilize this API. The `SAMLResponse` returned from the SAML v2 provider will be used to retrieve the user's email address.  FusionAuth if attempt to lookup by email and the user will be created if they do not yet exist.